footer.php

```diff
@@ -1,8 +1,50 @@
-  <footer>
-    <p>This is my footer.</p>
+</body>
+
+<footer>
+
+<div class="container-fluid text-center">
+  <div class="row align-items-center bg-secondary" style="footer">
+    <div class="col-sm-2">
+      <img src="<?php echo get_template_directory_uri() . '/assets/images/ClimateHubLogo.gif'; ?>" class="img-fluid border-0 chlogo"/>
+    </div>
+    <div class="col-sm-2">
+    <?php get_template_part( 'template-parts/footer_customfield' )?>
+    Creating a vibrant community around bold climate action at UBC.
+    </div>
+    <div class="col-sm-4 text-center mt-2 mb-2">
+
+      <h6 class="font-weight-bold ">FOLLOW US</h6>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

```

Comment:
Did you mean to link a font here?

Reply:
if you're talking about line 16...no, I just wanted it to be bold. I didn't realize that h6 had bold styling built in. I deleted * class="font-weight-bold " * 
or are you referring to line 17?

Reply:
I'm referring to line 17